### ISSUE TITLE
The Demo project does not work out of the box.

### ISSUE THREAD

**Issue by mcmurrym** [google/EarlGrey#125]
After doing a pod install, the demo project does not work. `grey_accessibilityID` seems to be the culprit which I also can't resolve when I add EarlGrey to my own project. This is making it extremely difficult to want to pursue when the setup is so rickety.

Swift Example:
![screen shot 2016-06-08 at 10 02 19 am](https://cloud.githubusercontent.com/assets/1041326/15901336/24590a08-2d60-11e6-827e-6b376954443d.png)

Objective-C Example:
![screen shot 2016-06-08 at 10 03 43 am](https://cloud.githubusercontent.com/assets/1041326/15901396/5a875db4-2d60-11e6-93ef-b8cbd01404a1.png)

**Comment by bootstraponline** [comment]
What version of OS X & Xcode are you using?

**Comment by mcmurrym** [comment]
OSX 10.11.5
Xcode 7.3

**Comment by khandpur** [comment]
Sorry that you're running into this Matt. @tirodkar: could you please help out Matt with this.

**Comment by khandpur** [comment]
@mcmurrym I think you might be running into:
http://stackoverflow.com/questions/37212028/use-of-unresolved-identifier-grey-accessibilityid/37215155#37215155

**Comment by mcmurrym** [comment]
I saw that, but just renaming the directory causes the BridgingHeader to fail on the import of `EarlGrey/EarlGrey.h`

**Comment by mcmurrym** [comment]
This is the example project in the error state before changing the directory name
[EarlGreyExample.zip](https://github.com/google/EarlGrey/files/305302/EarlGreyExample.zip)

**Comment by bootstraponline** [comment]
Try updating to the latest stable Xcode 7.3.1.

**Comment by tirodkar** [comment]
This seems to be related to a SourceKit issue with non allowing frameworks with hyphens or so in their names. An update might help in your case.

Try this - 

Rename Pods/EarlGrey/EarlGrey-1.0.0 to Pods/EarlGrey/EarlGrey1 and re-add the EarlGrey.framework in your project in the Pods/Pods/EarlGrey/Frameworks directory which should now be red on changing the directory name. Remove the Framework Search Path to EarlGrey in the Swift Target's Build Settings.

**Comment by mcmurrym** [comment]
It works just fine on Xcode 7.3.1 I recommend documenting this as a requirement. No directory renaming needed.

**Comment by tirodkar** [comment]
On it. Thanks.

**Comment by khandpur** [comment]
Closing as this fixed.
